hackage2nix/hackage2nix.cabal

```diff
@@ -30,7 +30,7 @@ executable hackage2nix
     , bytestring
     , Cabal >= 1.22.2
     , containers
-    , deepseq-generics
+    , deepseq
```

Comment:
We should probably depend on `deepseq >= 1.4`. Also, note that these Cabal files are generated from `package.yaml` by `stack`/`hpack`, so changes here will be lost.